Pipfile

```diff
@@ -13,6 +13,7 @@ thoth-analyzer = "*"
 prometheus-client = "*"
 pip = "*"
 toml = "*"
+package-extract = {path = "."}
```

Comment:
Will this break the image? I know installing the local package is sometimes necessary for the packages CLIs to be in `$PATH`